info/v2/container.go

```diff
@@ -166,3 +166,16 @@ type RequestOptions struct {
 	// Whether to include stats for child subcontainers.
 	Recursive bool `json:"recursive"`
 }
+
+type ProcessInfo struct {
+	User        string `json:"user"`
+	Pid         int    `json:"pid"`
+	Ppid        int    `json:"parent_pid"`
+	StartTime   string `json:"start_time"`
```

Comment:
Its actually convoluted to change the time fields back as the output changes based on the duration:

api/v2.0/ps/2ff16760cda11e8568589a9d73fe7e833b83ca3a8f55537157f7ff52195bad70?type=docker
[{"user":"root","pid":7003,"parent_pid":20768,"start_time":"May06","percent_cpu":"0.0","rss":"424","virtual_size":"7308","status":"Ss","running_time":"00:00:00","cmd":"stress"},{"user":"root","pid":7021,"parent_pid":7003,"start_time":"May06","percent_cpu":"99.5","rss":"262284","virtual_size":"269456","status":"R","running_time":"4-23:08:51","cmd":"stress"}]

start time can be May06, 20:11, or 2014. I think we can use a different option to print the whole time, but then that will be hard to condense back to be top like :/